docs/news.d/config_store.rst

```diff
@@ -0,0 +1,19 @@
+.. news-prs: 4162
+.. news-start-section: Additions
+- OpenDDS now stores ``rtps_udp_transport`` configuration in the key-value store.
+  The following members of ``RtpsUdpInst`` must now be accessed with getters and setters:
+
+  -  send_buffer_size_
+  -  rcv_buffer_size_
+  -  use_multicast_
+  -  ttl_
+  -  multicast_interface_
+  -  anticipated_fragments_
+  -  max_message_size_
+  -  nak_depth_
+  -  nak_response_delay_
+  -  heartbeat_period_
+  -  receive_address_duration_
+  -  responsive_mode_
```

Comment:
These should be quoted because they're symbol names. I don't care so much about that in a list like this and wasn't going say anything about this but [trailing underscores are part of the hyperlink syntax in RST](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets), so this broke the sphinx_strict check.

```suggestion
  -  ``send_buffer_size_``
  -  ``rcv_buffer_size_``
  -  ``use_multicast_``
  -  ``ttl_``
  -  ``multicast_interface_``
  -  ``anticipated_fragments_``
  -  ``max_message_size_``
  -  ``nak_depth_``
  -  ``nak_response_delay_``
  -  ``heartbeat_period_``
  -  ``receive_address_duration_``
  -  ``responsive_mode_``
```